### ISSUE TITLE
Spawing Entities Dynamically

### ISSUE THREAD

**Issue by rdelfin** [Trouv/bevy_ecs_ldtk#335]
Sorry if this is something that's mentioned elsewhere but I couldn't find any issues.

Is there currently a way of manually spawning a new entity based off an entity type in LDtk? To give an example, I have a game where entities spawn based on a specific event from specific spots on the map (the spawn points are themselves entities). What I'd like to do is use the "entity" objects in LDtk as a template to spawn a new entity. That way I can use the spritesheet as loaded in LDtk. Is this possible?

**Comment by TurtIeSocks** [comment]
+1 here, also interested. Or would be curious to hear if this is the wrong approach entirely and what would be a better way!

**Comment by rdelfin** [comment]
Yeah, that's a fair point! Maybe it's easier to have my own custom bundles and logic for doing that in game instead
